feature-libs/product-configurator/rulebased/components/overview-filter-button/configurator-overview-filter-button.component.html

```diff
@@ -0,0 +1,24 @@
+<ng-container *ngIf="config$ | async as config">
```

Comment:
@Uli-Tiger , @Larisa-Staroverova I think we need a loading animation here as well, do you agree? (Missing animation is very visible though, as the OV form loading animation is present)

Reply:
added a ghost animation